fars/booking/views.py

```diff
@@ -173,7 +175,8 @@ def delete(self, request, booking_id):
         is_admin = _is_admin(request.user, booking.bookable)
         if _is_admin or self.context['unbookable']:
```

Comment:
Not related to your PR, but should the condition be `is_admin or ...` instead of `_is_admin or ...`?

Reply:
The underscore is kind of a convention that is supposed to indicate a private method (since shitty python doesn't have those), but we might not be using the convention properly.

Reply:
I know of the convention, but the variable `is_admin` isn't used in the `if` straight after, but instead `_is_admin` is checked. Is that how it should be, won't it always render true?